docs/supported-technologies.asciidoc

```diff
@@ -262,6 +262,11 @@ The spans are named after the schema `<method> <host>`, for example `GET elastic
 |
 | 1.7.0
 
+|Apache Dubbo
+| 2.7.3 and upper,  2.6.x
```

Comment:
Why exactly 2.7.3? Does it not work with 2.7.{0,1,2}? What happens when the agent is used with versions lower than 2.6.0? Does it break, does it work? Should we try to avoid instrumenting those versions?

Reply:
> Why exactly 2.7.3? Does it not work with 2.7.{0,1,2}?

That's because the method `org.apache.dubbo.rpc.Result#whenCompleteWithContext` was not available previously. This means instrumenting a Dubbo 2.7.2 application would print lots of errors.

Reply:
The agent will not work with dubbo 2.7.{0,1,2}, since 2.7.3, dubbo provides a better async callback like  `AsyncRpcResult.whenCompleteWithContext`, the most importance is the 2.7.{0,1,2}   are not recommended in production by dubbo official， so the agent do not compatible with 2.7.{0,1,2}。

the agent may be work with versions lower than 2.6.0, but they have not been well tested.

Reply:
Thanks, I have tested with 2.5.x and it works. I've also ensured not to instrument unsupported versions. I think we are really close now! 🙂